src/algebra/algebra/subalgebra.lean

```diff
@@ -807,6 +807,10 @@ subtype.ext rfl
 @[simp] lemma coe_inclusion {S T : subalgebra R A} (h : S ≤ T) (s : S) :
   (inclusion h s : A) = s := rfl
 
+lemma inclusion_mk {E F : subalgebra R A} (hEF : E ≤ F) {x : A} (hx : x ∈ E) :
+  (subalgebra.inclusion hEF) ⟨x, hx⟩ = ⟨x, hEF hx⟩ :=
+rfl
```

Comment:
This looks to me like a stronger version of `inclusion_right`; can you delete that lemma and mark this one `simp`?